lib/puppet/provider/domain_name/nexus.rb

```diff
@@ -0,0 +1,25 @@
+require 'puppet/resource_api/simple_provider'
+require 'cisco_node_utils'
+
+class Puppet::Provider::DomainName::Nexus <  Puppet::ResourceApi::SimpleProvider
```

Comment:
and we just updated the device type to be `cisco_nexus`, so the provider should be called `::CiscoNexus` now.